backend/runner/observability/runner.go

```diff
@@ -0,0 +1,95 @@
+package observability
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/metric/noop"
+
+	ftlv1 "github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1"
+)
+
+const (
+	runnerMeterName              = "ftl.runner"
+	runnerDeploymentKeyAttribute = "ftl.deployment.key"
```

Comment:
Since this one isn't runner-specific, should we move it to internal/observability/attributes.go? We'll definitely want to use it for deploy/controller metrics too

Reply:
nice catch